docs/social_login.rst

```diff
@@ -111,8 +111,47 @@ prompted to accept the permissions requested:
 After accepting permissions, you'll be immediately redirected back to your
 website at the URL specified by ``redirectUrl`` in your app's config.
 
+.. note::
+
+  By default the configuration ``web.social.facebook.scope`` is set to ``email``. Change
+  this configuration value to request additional scopes.
+
 Simple, right?!
 
+Logging in using an Access Token
+................................
+
+If you want to use a Facebook SDK to login a user you can easily do so by grabbing the
```

Comment:
I really like your explanations at the beginning of the Google & LinkedIn sections. I would add that same explanation here if it applies. "Using the Access Token (implicit) workflow is preferable when you're logging in from a place where the storage can be trusted. E.g. from your back-end where only you have access to the Access Token."